x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview_logic.ts

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { HttpSetup } from 'src/core/public';
+
+import { kea } from 'kea';
+
+import { IAccount, IOrganization, IUser } from '../../types';
+import { IFlashMessagesProps, IKeaLogic, TKeaReducers, IKeaParams } from '../../../shared/types';
+
+import { IFeedActivity } from './recent_activity';
+
+export interface IOverviewServerData {
+  hasUsers: boolean;
+  hasOrgSources: boolean;
+  canCreateContentSources: boolean;
+  canCreateInvitations: boolean;
+  isOldAccount: boolean;
+  sourcesCount: number;
+  pendingInvitationsCount: number;
+  accountsCount: number;
+  personalSourcesCount: number;
+  activityFeed: IFeedActivity[];
+  organization: IOrganization;
+  isFederatedAuth: boolean;
+  currentUser: IUser;
+  fpAccount: IAccount;
+}
+
+export interface IOverviewActions {
+  setServerData(serverData: IOverviewServerData): void;
+  setFlashMessages(flashMessages: IFlashMessagesProps): void;
+  setHasErrorConnecting(hasErrorConnecting: boolean): void;
+  initializeOverview({ http }: { http: HttpSetup }): void;
+}
+
+export interface IOverviewValues extends IOverviewServerData {
+  dataLoading: boolean;
+  hideOnboarding: boolean;
```

Comment:
```suggestion
```

Also one more instance of `hideOnboarding` to delete

Reply:
Sorry about those 🤦

Reply:
Haha, all good!